src/Core/MakeModelMenu.elm

```diff
@@ -0,0 +1,350 @@
+module Core.MakeModelMenu exposing (processRemoteData, getAvailableMakes, getAvailableModels, modelsApiUrl, makesApiUrl, main, init, update, subscriptions, view, Flags, Model, Msg)
```

Comment:
Do all of these need to be exposed? Shouldn't only `main` be needed?

Reply:
Actually, we shouldn't have main I think, because it needs to use ports etc... it will need to expose init, update, subscriptions, model and then the using app will integrate the ports (see notifications)